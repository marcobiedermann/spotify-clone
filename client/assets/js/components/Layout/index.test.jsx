import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import { Layout } from '.';

it('renders correctly', () => {
  const tree = renderer
    .create(<MemoryRouter><Layout /></MemoryRouter>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
