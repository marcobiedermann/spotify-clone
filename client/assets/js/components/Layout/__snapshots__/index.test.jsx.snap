// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="layout"
>
  <header>
    <form
      className="search"
      onSubmit={[Function]}
    >
      <input
        className="search__input"
        type="search"
      />
    </form>
    <div
      className="me"
    >
      <figure
        className="me__image"
      />
      <a
        href="/users/"
        onClick={[Function]}
      >
        
      </a>
    </div>
  </header>
  <div
    className="content"
  >
    <main
      className="main"
    />
    <aside
      className="aside aside--left"
    >
      <nav>
        <ul>
          <li>
            <a
              href="/"
              onClick={[Function]}
            >
              Home
            </a>
          </li>
          <li>
            <a
              href="/browse"
              onClick={[Function]}
            >
              Browse
            </a>
          </li>
        </ul>
      </nav>
      <ul />
    </aside>
  </div>
  <footer>
    Footer
  </footer>
</div>
`;
